src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -65,6 +65,8 @@ namespace winrt::TerminalApp::implementation
         std::optional<int> _rearrangeFrom;
         std::optional<int> _rearrangeTo;
 
+        ShortcutActionDispatch _actionDispatch{};
```

Comment:
```suggestion
        ShortcutActionDispatch _actionDispatch;
```
Should we initialize here or in the constructor? I honestly don't know! But I'll point it out haha

Reply:
I think here's fine - I think we're supposed to be moving towards in-class initialization as opposed to ctor initialization. I think I prefer the explicit initialization here, since this is a winrt type that should be explicitly constructed to _not_ `nullptr`